### DISCUSSION TITLE
Can not submit request  for external series in navigation UI.

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [dcm4che/dcm4chee-arc-light#4074] [General]
When I config a remote AE , 

it is ok :
http://local_address/dcm4chee-arc/aets/Local_AE/dimse/EXT_AE/studies?limit=21&includefield=all&offset=0

it is not ok. 
http://local_address/dcm4chee-arc/aets/Local_AE/dimse/EXT_AE/series?limit=21&includefield=all&offset=0

Not sure whether is a bug or only study level is allowed to submit for remote AE.

**Comment by luxunxiansheng** [comment]
Any more comments?

**Comment by vrindanayak** [comment]
> it is not ok.
http://local_address/dcm4chee-arc/aets/Local_AE/dimse/EXT_AE/series?limit=21&includefield=all&offset=0

Can you elaborate - did you get some error or do you see something in server log?
[Search for Series from external C-FIND SCP](https://petstore.swagger.io/index.html?url=https://dcm4che.github.io/dcm4chee-arc-light/swagger/openapi.json#/FIND-RS/SearchForSeriesExternal) REST API is supported since version 5.29.1 - https://github.com/dcm4che/dcm4chee-arc-light/issues/3882
